crates/evm/src/fuzz/mod.rs

```diff
@@ -301,25 +301,33 @@ impl BaseCounterExample {
         contracts: &ContractsByAddress,
         traces: Option<CallTraceArena>,
     ) -> Result<Self> {
-        let (name, abi) = &contracts.get(&addr).ok_or(FuzzError::UnknownContract)?;
-
-        let func = abi
-            .functions()
-            .find(|f| f.short_signature() == bytes.0.as_ref()[0..4])
-            .ok_or(FuzzError::UnknownFunction)?;
-
-        // skip the function selector when decoding
-        let args =
-            func.decode_input(&bytes.0.as_ref()[4..]).map_err(|_| FuzzError::FailedDecodeInput)?;
+        if let Some((name, abi)) = &contracts.get(&addr) {
```

Comment:
this now always returns Ok?

is this intended?

Reply:
Yes, was always returning Ok. Removed unnecessary Result. Just returns Self now.